.github/workflows/process-fbc-fragment.yaml

```
@@ -0,0 +1,72 @@
+              PREV_PATCH=0
+              PREV_VERSION="$PREV_MAJOR.$PREV_MINOR.$PREV_PATCH"
+            else
+              PREV_PATCH=0
+              PREV_MINOR=$((MINOR - 1))
+              PREV_VERSION="$MAJOR.$PREV_MINOR.$PREV_PATCH"
+            fi
+          else
+            PREV_PATCH=$((PATCH - 1))
+            PREV_VERSION="$MAJOR.$MINOR.$PREV_PATCH"
+          fi
+        make catalog-prepare -e BUNDLE_IMGS=${QUAY_IMG}:v${PREV_VERSION},${QUAY_IMG}:v${CURRENT_VERSION}
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 60: could not find expected ':'

(syntax-check)